### PR TITLE
Added a code of conduct

### PR DESCRIPTION
**Pull request by Twister915**

### CODE REVIEW

**Comment by Jake0oo0** [comment]
:+1: All projects should include a code of conduct in order to prevent hateful comments against those that intend to contribute to the project. Something like "Joey is a shit designer" should be removed as a violation of these conduct rules, despite it being a true statement. It also provides protection for gays, lesbians, bisexuals, transgenders, large people, short people, non-white people, and other minorities that may choose to contribute to the Trident SDK project. I believe that this is a very important step forward for Trident and its contributors and will provide a safe place for those that wish to contribute to the project's future success. 

Just kidding, fuck Github and their shitty code of conduct.

**Comment by Twister915** [comment]
@Jake0oo0 You should be more professional, and follow the code of conduct. @agelender ^

**Comment by AgentTroll** [comment]
"constructive and never resort to personal attacks, trolling, public or private harrassment, insults, or other *unprofessional conduct*"

TL;DR Respect the English language :)

**Comment by Twister915** [comment]
https://github.com/angular/code-of-conduct/blob/master/CODE_OF_CONDUCT.md

**Comment by AgentTroll** [comment]
Closed. TridentSDK is an open, friendly community. We want all opinions heard and stated, from individuals with kind and constructive minds, to those whose intentions are to muddling our project. 

As appropriate, those who feel insecure or subject to any type of harassment as a result of being a part of TridentSDK, whether it be a member of the community or a contributor to the project, whether it be a person we like or don't like, and finally, whether it be sourced from another member, one of the core developers (as I have experienced in other projects), they may report it to us privately. Vilsol has setup an email at support@tridentsdk.net (of questionable service status, may I add), and I may be privately referred to at pierrec.tridetnsdk@gmail.com. 

Among that, we do not have a conduct email domain, it has been submitted to the implementation project (which we'd like to have files of the like submitted to TridentSDK instead), and not everyone can agree to the code of conduct. We expect all members to abide by a general set of moral rules. 

And again, those who are subject to disrespect of **any** form (bullying, harassment, discrimination, etc...), from **any** source (another member, a developer, a staff member, etc...), as a result of being a pet of the TridentSDK ecosystem (forums, Twitter responses, GitHub, etc...) can speak to use as described the previous 2 paragraphs.

If you would like to further the discussion of this PR to a broader audience, please post a thread in the Trident Development section of our forums.
